installer/driverInstall.js

```diff
@@ -32,7 +32,10 @@ if((process.env.npm_config_vscode)||(__dirname.toLowerCase().indexOf('db2connect
         vscodeVer = parseFloat(codeOut.split('\n')[0]);
 
         if(!isNaN(vscodeVer)){
-            if(vscodeVer >= 1.30){
+			if(vscodeVer >= 1.36){
```

Comment:
@akhilravuri1 Correct the indentation here. Tab should be replaced by spaces.

Reply:
@bimalkjha I have corrected the indentation. Can you please check it.